### ISSUE TITLE
Documentation: Autoincrement problem in part 2 of tutorial

### ISSUE THREAD

**Issue by sevanspowell** [tathougies/beam#258]
I believe the auto increment keyword used in this part of tutorial 2:

```
sqlite> CREATE TABLE addresses ( id INT AUTO_INCREMENT PRIMARY KEY, ...
```

should be:

```
sqlite> CREATE TABLE addresses ( id INTEGER PRIMARY KEY AUTOINCREMENT, ...
```

Or something similar, otherwise I don't get an auto-incrementing id column in the address table.

**Comment by sevanspowell** [comment]
Submitted PR #259.

**Comment by sevanspowell** [comment]
Already fixed with 4b1fdd7.
